<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotDialog</class>
 <widget class="QDialog" name="PlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plotar Gráfico</string>
  </property>
  <widget class="QCheckBox" name="count_request_cname">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>count_request_cname</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="count_request_not_in_whitelist">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>count_request_not_in_whitelist</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="std_domain_length">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>std_domain_length</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="count_request_mx">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>count_request_mx</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="minimum_requisition_degree">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>minimum_requisition_degree</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="std_requisition_degree">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>std_requisition_degree</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="count_request_txt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>count_request_txt</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="average_requisition_degree">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>average_requisition_degree</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="count_request_a">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>count_request_a</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="average_domain_length">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>average_domain_length</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="count_request">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>count_request</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="count_domain_with_numbers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>count_domain_with_numbers</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecione 2 ou 3 características</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>criar gráfico</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
